explorer/mempool.go

```diff
@@ -386,3 +387,41 @@ func (votes byHeight) Len() int {
 func (votes byHeight) Swap(i, j int) {
 	votes[i], votes[j] = votes[j], votes[i]
 }
+
+// matchMempoolVins filters relevant mempool transaction inputs
+// whose previous outpoints match the specified transaction id.
+func matchMempoolVins(txid string, txsList []MempoolTx) (vins []MempoolVin) {
+	for idx := range txsList {
+		tx := txsList[idx]
```

Comment:
`MempoolTx`  is a large structure, and it wouldn't hurt to instead do `tx := &txsList[idx]`.

Reply:
But if we really wanted a copy, not a pointer, it would be better to do `for _, tx := range txsList {` since `idx` is used nowhere else.